js/fileinput.js

```diff
@@ -1027,6 +1003,8 @@
             if (self.hideThumbnailContent) {
                 $h.addCss(self.$preview, 'hide-content');
             }
+            $cont.find('.file-caption-name').on('keydown', function () { return false; })
```

Comment:
This handles `onkeydown` and `onpaste` event while `bootstrap-fileinput` is loaded on the DOM since it is in `FileInput.prototype._init` function.

Did I miss something ?